mmdet/core/mask/structures.py

```diff
@@ -498,6 +554,30 @@ def crop_and_resize(self,
             resized_masks.append(resized_mask)
         return PolygonMasks(resized_masks, *out_shape)
 
+    def translate(self,
+                  out_shape,
+                  offset,
+                  direction='horizontal',
+                  fill_val=0,
+                  interpolation='bilinear'):
+        """Translate the PolygonMasks."""
```

Comment:
We can set `fill_val` and `interpolation` here to be None. Report a warning when the user pass other values when translating a polygon mask since it doesn't support.